dozer-sink-oracle/src/lib.rs

```diff
@@ -246,10 +246,10 @@ impl OracleSinkFactory {
         for field in &schema.fields {
             let name = &field.name;
             let col_type = match field.typ {
-                dozer_types::types::FieldType::UInt => "NUMBER",
-                dozer_types::types::FieldType::U128 => unimplemented!(),
-                dozer_types::types::FieldType::Int => "NUMBER",
-                dozer_types::types::FieldType::I128 => unimplemented!(),
+                dozer_types::types::FieldType::UInt => "INTEGER",
+                dozer_types::types::FieldType::U128 => "INTEGER",
```

Comment:
Does `INTEGER` have a large enough precision? I thought it had precision `22`, which is only enough for about 70 bits, I believe

Reply:
From my understanding, oracle can guarantee precision up to 38.